docs/blog/2019-02-26-getting-started-with-gatsby-themes/index.md

```diff
@@ -149,14 +132,14 @@ module.exports = {
 
 Now, you can make sure _site_ is linked to _theme_.
```

Comment:
I would move this under "### Setting up Site `package.json`" as we higlight on how to setup the theme inside package.json